<?xml version="1.0"?>
<launch>
    <node name="object_detector" pkg="object_detection" type="object_detector.py">
        <rosparam command="load" file="$(find object_detection)/param/object_detector.yaml" />
    </node>
</launch>
